common/src/main/java/earth/terrarium/adastra/api/events/AdAstraEvents.java

```diff
@@ -43,6 +47,26 @@ static boolean post(ServerLevel level, LivingEntity entity) {
         }
     }
 
+    @FunctionalInterface
+    public interface EntityHasOxygenEvent {
```

Comment:
For consistency, this can just be called `EntityOxygenEvent`. None of the other events have prefixes like that. and `ENTITY_HAS_OXYGEN_LISTENERS` should be renamed to `ENTITY_OXYGEN_LISTENERS` as well.

Reply:
changed